src/Auth/FootprintAwareTrait.php

```diff
@@ -61,7 +61,8 @@ public function footprint(Event $event)
         }
 
         if ($event->name() === 'Auth.afterIdentify') {
-            $this->_listener->setUser($this->_getCurrentUser($event->data));
+            if (is_array($event->data))
```

Comment:
I noticed that this has been triggered tow times the first was with  `$event->data = null`